atef/widgets/config/data_active.py

```diff
@@ -652,6 +653,14 @@ def update_all_desc(self, *args, **kwargs):
             row_widget: CheckRowWidget = self.checks_table.cellWidget(ind, 0)
             row_widget.update_summary()
 
+    def nav_to_self(self):
+        # A bit of a hack to prevent navigating to details prematurely
+        # normally an isinstance check would be good, but circular imports
+        try:
+            self.parent().parent().full_tree.select_by_data(self.data)
```

Comment:
Briefly scrutinizing this: why is the parent's parent always the instance that owns the full tree?

Reply:
The `SetValueStep` widget is inside a QWidget container, which is itself inside a `StepPage(PageWidget)` widget.  only those `PageWidget` classes can see the full tree